themes/bootstrap3/templates/search/results.phtml

```diff
@@ -83,12 +77,13 @@
 
   <?php if ($recordTotal < 1): ?>
     <p>
-      <?php if (isset($this->overrideEmptyMessage)): ?>
-        <?=$this->overrideEmptyMessage?>
-      <?php else: ?>
-        <?php $this->layout()->srmessage = $this->translate('nohit_lookfor_html', ['%%lookfor%%' => $this->escapeHtml($lookfor)]); ?>
-        <?=$this->layout()->srmessage ?>
-      <?php endif; ?>
+      <?php
+        $emptyMessage = $this->slot('empty-message')->get(
+          $this->translate('nohit_lookfor_html', ['%%lookfor%%' => $this->escapeHtml($lookfor)])
+        );
+        $this->layout()->srmessage = $emptyMessage;
```

Comment:
Another functional change -- previously, the screen reader message was always nohit_lookfor_html, but now it can be overridden. Perhaps that's a bug fix, but might be worth re-examining why we are doing this.

Reply:
I think the bug here was that the overrideEmptyMessage wasn't put forward as the sr-only message.